terraform/modules/platform/README.md

```diff
@@ -14,30 +14,54 @@ The key assumptions are focused on the existence of resources that managed exter
 
 ## Example Usage
 
+
+1. To use data and ssm parameters, import the cdap platform module.
 ```hcl
 ## AB2D API Module Usage Example
 module "platform" {
-  # Ensure `ref` in the following line is pinned to something static
-  # e.g. a known branch, commit hash, or tag from **this repository**
-  source = "git::https://github.com/CMSgov/cdap.git//terraform/modules/platform?ref=plt-1033"
-
-  app         = "ab2d"
-  env         = var.env
-  root_module = "https://github.com/CMSgov/ab2d-ops/tree/main/terraform/services/api"
-  service     = "api"
+  source    = "git::https://github.com/CMSgov/ab2d-bcda-dpc-platform.git//terraform/modules/platform?ref=PLT-1099"
+  providers = { aws = aws, aws.secondary = aws.secondary }
+
+  app          = local.app
+  env          = local.env
+  root_module  = "https://github.com/CMSgov/ab2d/tree/main/ops/services/20-microservices"
+  service      = local.service
+  ssm_root_map = local.ssm_root_map
```

Comment:
The "local" references in this usage example are not so helpful because we don't know what the values are. It would be better to have example values like "ab2d", "test", "api", etc.

An example for `ssm_root_map` would be especially helpful. I still don't know what it would look like.